test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -62,6 +62,24 @@ public async Task GivenMetadata_WhenRead_ThenAuditLogEntriesShouldNotBeCreated()
             Assert.Empty(_auditLogger.GetAuditEntriesByCorrelationId(correlationId));
         }
 
+        [Fact]
```

Comment:
This should be a skippable fact and use the skip function so we know it was skipped and not run successfully.